### PR TITLE
Update PNI product page analytics

### LINKED ISSUE #5239: Update analytics for PNI - Product pages
- [x] define the list of events and which UI elements and user actions they are tied to
- [x] define the list of old events that will need to be removed

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5239

Spreadsheet: https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit?usp=sharing

Sample test page: https://foundation-s-issue-5239-enideu.herokuapp.com/en/privacynotincluded/products/percy-cypress/

### CODE REVIEW

**Review by taisdesouzalessa — CHANGES_REQUESTED** [review]
Hey Mavis,

I tested it and found one item that isn't working as expected. **See below**. 

For the action "security expand accordion tap" the name of the product is displaying as "Undefined"

image


Also, I have some comments for 2 things I couldn't test (multiple privacy policy link tap and update article link tap) plus 3 new items I forgot to include in the first iteration (sorry). All of the comments are in our list (column G) for better context: https://docs.google.com/spreadsheets/d/1x-CP_HYaAs3oRv2cjbSFu8w5Z938E4cyHQ3hvh17c6c/edit?usp=sharing

**Comment by mmmavis** [comment]
@taisdesouzalessa thanks for the notes. I'll address those issues soon.

> Also, I have some comments for 2 things I couldn't test (multiple privacy policy link tap and update article link tap)

I can update the example test page. Just a reminder that you can always log in to CMS and add/edit content to suit your testing needs 😉

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -263,7 +263,7 @@ <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
       <div class="row">
         <div class="{{section_class}} mt-5 pt-3 bg-white product-section-padding-x">
           {% if product.updates|length > 0 or coralTalkServerUrl %}
-          <h3 class="pni-product-smaller-body-bold mb-4 mb-md-5">{% trans "Updates" %}</h3>
+          <h3 class="h2-heading mb-4 mb-md-5">{% trans "Updates" %}</h3>
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/product_page.html:266]: This line was accidentally updated by me in my other PR. It should be using `h2-heading`.

**Comment by mmmavis** [comment]
PR updated!

**Comment by taisdesouzalessa** [comment]
> I can update the example test page. Just a reminder that you can always log in to CMS and add/edit content to suit different testing needs 
ahhh true, I keep forgetting that I can edit the CMS. Sorry Mavis! Maybe there is a CMS cheatsheet or something I can do to remember, it is too much to keep in mind...haha.

**Review by taisdesouzalessa — APPROVED** [review]
all good! Thanks, Mavis \0/

**Review by Pomax — APPROVED** [review]

**Pull request merged**